src/main/scala/mesosphere/marathon/core/matcher/reconcile/impl/OffersWantedForReconciliationActor.scala

```diff
@@ -7,13 +7,14 @@ import java.time.Clock
 import akka.actor.{Actor, Cancellable, Props}
 import akka.event.{EventStream, LoggingReceive}
 import com.typesafe.scalalogging.StrictLogging
-import mesosphere.marathon.core.flow.ReviveOffersConfig
 import mesosphere.marathon.core.event.DeploymentStepSuccess
 import mesosphere.marathon.state.Timestamp
 import mesosphere.marathon.core.deployment.StopApplication
+import mesosphere.marathon.core.launchqueue.ReviveOffersConfig
 
 import scala.concurrent.duration._
 
+// TODO: this actor works against the ReviveOffersActor and needs to be adjusted/removed
```

Comment:
If I'm not mistaken we need the revives for the `OfferMatcherReconciler` which should be refactored anyways for [MARATHON-8517](https://jira.mesosphere.com/browse/MARATHON-8517).

When is this actor triggered? Maybe be can fold the logic into the new `ReviveOffersActor`.